lib/jni/LogManager_jni.cpp

```diff
@@ -869,10 +869,28 @@ Java_com_microsoft_applications_events_LogManagerProvider_00024LogManagerImpl_na
     jstring jSource,
     jstring jScope)
 {
+    if (!thiz) {
```

Comment:
Should it be (!this) not sure if `thiz` is a typo.

Reply:
I'm surprised this didn't break the build

Reply:
thiz is the JNI conventional argument for the "this" Java pointer (jobject), because C++ dislikes it if you decide that some random variable should be named "this."